modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl

```diff
@@ -45,10 +45,20 @@
 		display: block;
 	}
 
+	.developer-name {
+		color: #545d69;
+	}
+
 	.productSpec {
 		color: #545d69;
 	}
 
+	.solution-search-image {
```

Comment:
Provide more context for this selector. If the style is specific to an element, I need to make sure that the style is only applied to that element.

```suggestion
	.adt-solutions-search-results .solution-search-results-card .solution-search-image {
```